ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/wan_path_groups.schema.yml

```
@@ -49,3 +49,17 @@ keys:
               local:
                 type: str
                 description: Optional, if not set, the path-group `name` is used as local.
+        dps_keepalive_interval:
```

Since we have interval below, I think we should avoid having interval in this key. I made it plural here, but it is up to you if it should be `dps_keepalive` or `dps_keepalives`.
```suggestion
        dps_keepalives:
```